### ISSUE TITLE
graphql-input provides to Customer as many errors as appeared instead of last one like on Magento Storefront

### ISSUE THREAD

**Issue by TomashKhamlai** [magento/graphql-ce#416]
### Preconditions (*)

1. Any supported product with more then 1 required customizable options

### Steps to reproduce (*)

1. Add product to cart without providing options

### Expected result (*)

1. Screenshot:
![image](https://user-images.githubusercontent.com/11827230/53569967-03efcf80-3b6e-11e9-81b4-fc74a70444ff.png)

2. GraphQL:
```
{
  "errors": [
    {
      "message": "The product's required option(s) weren't entered.
Make sure the options are entered and try again.",
      "category": "graphql-input",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "addSimpleProductsToCart"
      ]
    }
  ],
  "data": {
    "addSimpleProductsToCart": null
  }
}
```

### Actual result (*)

1. GraphQL:
```
{
  "errors": [
    {
      "message": "The product's required option(s) weren't entered.
Make sure the options are entered and try again.\n
The product's required option(s) weren't entered.
Make sure the options are entered and try again.",
      "category": "graphql-input",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "addSimpleProductsToCart"
      ]
    }
  ],
  "data": {
    "addSimpleProductsToCart": null
  }
}
```

If it is the feature then it's worth to document the benefits of such behaviour with usage example. Guidelines or devdocs are good places for this.

**Comment by lenaorobei** [comment]
Please use https://github.com/magento/graphql-ce/pull/421 as a reference. Need to rework the PR and use `Magento\Framework\Exception\AggregateExceptionInterface` for this implementation.

**Comment by paul-stolk-webdiensten** [comment]
@magento I am working on it

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for your report. 
The issue has been fixed in magento/graphql-ce#1023 by @paul-stolk-webdiensten in 2.3-develop branch
Related commit(s):
 - [18fb48296860b0fad99c80a44995f104f550f42c](https://github.com/magento/graphql-ce/commit/18fb48296860b0fad99c80a44995f104f550f42c)
 - [de92f124ede8a6d3b2dcd3f1c854b0c661c57ed1](https://github.com/magento/graphql-ce/commit/de92f124ede8a6d3b2dcd3f1c854b0c661c57ed1)
 - [4a8931227f3c45557eae04693d362e276476002e](https://github.com/magento/graphql-ce/commit/4a8931227f3c45557eae04693d362e276476002e)
 - [cbeb99ca2e35a173ad1b67c4a76bddf263ab4197](https://github.com/magento/graphql-ce/commit/cbeb99ca2e35a173ad1b67c4a76bddf263ab4197)

The fix will be available with the upcoming 2.3.5 release.
